MeetupPlatformApi/Domain/Meetup.cs

```diff
@@ -11,4 +13,8 @@ public class Meetup
     public DateTime EndTime { get; set; }
 
     public string Description { get; set; }
+
+    public Guid OrganizerId { get; set; }
+
+    public List<PlainUser> Users { get; set; }
```

Comment:
We don't add this column to database, therefore name of property makes no sense.
What about code, property contains user entities and 'users' name is best for property.

Reply:
yes, but who are these users in fact? they are some kind of meetup visitors and it seems to me that it would be more correct to address them as
Meetup.Visitors than Meetup.Users
because Users does not provide a full description of who we are calling, perhaps it is some kind of speakers, visitors, organizers, or is it a generalized field for visitors and speakers, for example
but I'm not sure about it and let it be so you're right